axis_backend/apps/clients/repositories/industry_repository.py

```
@@ -0,0 +1,146 @@
+"""Repository for Industry model data access."""
+from typing import Optional, List
```

Ruff (0.14.5)

`typing.List` is deprecated, use `list` instead

(UP035)